[2023-09-22T16:52:10.463+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-22T16:52:10.607+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.greet__1 scheduled__2021-11-09T00:00:00+00:00 [queued]>
[2023-09-22T16:52:10.627+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.greet__1 scheduled__2021-11-09T00:00:00+00:00 [queued]>
[2023-09-22T16:52:10.629+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 6
[2023-09-22T16:52:10.660+0000] {taskinstance.py:1350} INFO - Executing <Task(_PythonDecoratedOperator): greet__1> on 2021-11-09 00:00:00+00:00
[2023-09-22T16:52:10.666+0000] {standard_task_runner.py:57} INFO - Started process 442 to run task
[2023-09-22T16:52:10.672+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'greet__1', 'scheduled__2021-11-09T00:00:00+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/test2.py', '--cfg-path', '/tmp/tmpokve0tz5']
[2023-09-22T16:52:10.679+0000] {standard_task_runner.py:85} INFO - Job 88: Subtask greet__1
[2023-09-22T16:52:10.711+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:188 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-22T16:52:10.766+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_with_taskflow_api_v02.greet__1 scheduled__2021-11-09T00:00:00+00:00 [running]> on host 75cee4990e54
[2023-09-22T16:52:10.785+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-22T16:52:11.175+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v02' AIRFLOW_CTX_TASK_ID='greet__1' AIRFLOW_CTX_EXECUTION_DATE='2021-11-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-11-09T00:00:00+00:00'
[2023-09-22T16:52:11.177+0000] {logging_mixin.py:149} INFO - Hello World! My name is Jerry Fridman and I am 19 years old!
[2023-09-22T16:52:11.179+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-22T16:52:11.189+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v02, task_id=greet__1, execution_date=20211109T000000, start_date=20230922T165210, end_date=20230922T165211
[2023-09-22T16:52:11.220+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-09-22T16:52:11.234+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
